MiniTeks/mks-server
# Add controller for MksTaskRun

# Add controller for MksTaskRun

- Listens for MksTaskRun resource being created, updated or deleted
- Adds corresponding TaskRun using Tekton API

# How to test

- Build and run the executable using steps in README.md
- Apply the CRDs and example in the config folder.
- Add  a new MksTaskRun resource and see for logs

# Related issues

Fixes #13
